### DISCUSSION TITLE
Login with device cannot be used to unlock vault on page refresh

### DISCUSSION THREAD

**Discussion by gamedevsam** [dani-garcia/vaultwarden#5101] [Ideas]
Hello, thanks for this amazing piece of open source software. I setup "Login with Device" and everything worked smoothly after I enabled 'Approve login requests' as well as 'Allow sync on refresh' in the mobile app ([source](https://www.reddit.com/r/Bitwarden/comments/zt2ce8/comment/j1bldqz/)).

However after playing around with the feature a bit I noticed an opportunity for improvement. After my session expires on the web client (if I refresh the page for example), I am prompted to re-renter my Master Password to login, but there is no button to Login with Device.

Would it be possible to allow Login with Device when your the system recognizes you're attempting to re-login?

PS: Might not be a bad idea to add a section in the Wiki to explain the settings you need to enable on the client to get this feature working.

**Comment by BlackDex** [comment]
This is a client feature, and those are not maintained or developed by this project.
All client features are done by Bitwarden. And Vaultwarden tries to be compatible with these clients.

P.S. The wiki can be edited by everyone, so feel free to improve ours.

**Reply by gamedevsam** [reply]
Ohh, I didn't realize the web UI was also developed by Bitwarden. Good to know!
